### ISSUE TITLE
Mac: Close tab buttons missing

### ISSUE THREAD

**Issue by Martii** [QupZilla/qupzilla#2328]
With Qt 5.9 nearing release status from https://wiki.qt.io/Qt_5.9_Release thought I would make this issue to see what is up.

Comparisons:
1. 2.1.2 with Qt 5.8
2. 54949f94f3d309e633de671205a1691040840cc3 with Qt 5.9 Beta 2 and Beta 3

---

1. <img src="https://cloud.githubusercontent.com/assets/114709/25764042/41757156-31a3-11e7-997e-9c8fe96e6d3d.png" alt="2.1.2withqt5.8]" title="Click to enlarge" width="160" height="92" />
2. <img src="https://cloud.githubusercontent.com/assets/114709/25764091/73ee9734-31a3-11e7-861f-8407aa2ee66c.png" alt="54949f94f3withqt5.9beta3" title="Click to enlarge" width="160" height="92" />

**Comment by r-a-v-a-s** [comment]
Confirmed.

**Comment by Martii** [comment]
> They seem to still be there but invisible.

Yup... no solution yet... Qt 5.9 release didn't resolve it nor did rc1 and rc1.

**Comment by nowrep** [comment]
What about with Windows theme? Do you see a close button then?

**Comment by Martii** [comment]
> What about with Windows theme? Do you see a close button then?

Yes.

**Comment by Martii** [comment]
Linux and Chrome theme is missing them too.

**Comment by Martii** [comment]
Initial start up screen shot with cc01232 ... not sure what that empty tab is but it did close after I figured you pushed the close button to the right instead of left *(briefly saw that in commit)*.

<a href="https://user-images.githubusercontent.com/114709/36310556-623da656-1320-11e8-9e74-c1ea430d63f9.png"><img src="https://user-images.githubusercontent.com/114709/36310556-623da656-1320-11e8-9e74-c1ea430d63f9.png" alt="startup at " title="Click to enlarge" width="168" height="105"/></a>

Windows theme is still the only one that shows an "X" for closing. All other themes it is absent.

**Comment by Martii** [comment]
Correction... comes back on open *(relaunch of QupZilla)*... disappears after new tab click and then close that new tab.

**Comment by Martii** [comment]
@nowrep 
Do you really want me to release a tab bugged version for OS X here for 2.2.6  e.g. it's probably going to generate some noise that's already been mentioned here by me.

**Comment by nowrep** [comment]
You can copy the css (and images) that changes close button from Windows theme to all other themes to fix it.

**Comment by Martii** [comment]
The "blank space" before the tab is present is present on the Windows theme as of 879dce4 ... so not sure that would help.

**Comment by Martii** [comment]
Here's a video at current HEAD.

[tab_issue.zip](https://github.com/QupZilla/qupzilla/files/1821928/tab_issue.zip "HEVC ~908.0KiB")

**Comment by nowrep** [comment]
Well, it's not the only thing that is broken on macOS.This is just a visual bug - current tab is drawn in wrong place.
